index.html

```diff
@@ -329,7 +332,7 @@ <h1 class="exp_main">
       <br>
     </section>
     <hr class="divide">
-    <section>
+    <div class="socials">

```

Comment:
**- You can try to fix this using display flex.**

![Screen Shot 2021-07-22 at 05 00 57](https://user-images.githubusercontent.com/22258274/126622907-6b6b5ede-6628-474e-966f-c8df6dfa87e2.png)